.github/workflows/package.yml

```diff
@@ -67,8 +67,26 @@ jobs:
             autoconf-archive \
             devscripts \
             python3-pip \
-            qemu-user-static \
             ruby
+      - name: Install Qemu from Groovy
```

Comment:
```suggestion
      - name: Install QEMU from Groovy
```

Reply:
Could you add a comment that describes why we need this and when we can remove this?

Reply:
@komainu8 Did you miss https://github.com/groonga/groonga/pull/1176#discussion_r612981895 ?

Reply:
No. I confirm it.
I'm fixing now. Please wait a little longer.